surfsense_backend/alembic/versions/11_add_llm_config_table_and_relationships.py

```
@@ -0,0 +1,86 @@
+"""Add LLMConfig table and user LLM preferences
+
+Revision ID: 11
+Revises: 10
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import UUID, JSON
```

Ruff (0.11.9)

`sqlalchemy.dialects.postgresql.UUID` imported but unused

Remove unused import

(F401)